src/admin/class-admin.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * The Admin specific functionality.
+ *
+ * @since   4.0.0 Major refactor.
+ * @since   1.0.0
+ * @package Inf_Theme\Admin
+ */
+
+namespace Inf_Theme\Admin;
+
+use Eightshift_Libs\Core\Service;
+
+use Inf_Theme\Helpers\General_Helper;
+
+/**
+ * Class Admin
+ *
+ * This class handles enqueue scripts and styles and some
+ * admin facing functionality.
+ */
+class Admin implements Service {
+
+  /**
+   * Register all the hooks
+   *
+   * @return void
+   *
+   * @since 4.0.0 Adding Register hooks.
+   * @since 1.0.0
+   */
+  public function register() : void {
+    \add_action( 'login_enqueue_scripts', [ $this, 'enqueue_styles' ] );
```

Comment:
Just wondering, should we force load from global namespace. Could become difficult to mock if we decide to later on (just thinking out loud). If it doesn't throw an error when we run it (just `add_action` instead of `\add_action`), we can use the unslashed version.

Reply:
👍